cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -1074,4 +1081,15 @@ private ProgramSpecification getProgramSpecificationWithoutAuthz(ProgramId progr
                                                     plugin.getPluginClass().getRequirements()))
       .collect(Collectors.toSet());
   }
+
+  private void authorizePipelineRuntimeImpersonation(Map<String, String> userArgs) throws Exception {
+    if ((userArgs.containsKey(SystemArguments.RUNTIME_PRINCIPAL_NAME)) &&
+            (userArgs.containsKey(SystemArguments.RUNTIME_KEYTAB_PATH))) {
+        String principal = userArgs.get(SystemArguments.RUNTIME_PRINCIPAL_NAME);
+        LOG.debug("Checking authorisation for user: %s, using runtime config principal: %s",
+            authenticationContext.getPrincipal(), principal);
```

Comment:
Wrong alignment. Multiple lines alignment should be like this:

```java
LOG.debug("......",
          authenticxationContext...);
```

Reply:
Done